.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/buildpack-deps:14.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: test
+          command: docker-compose run --rm app rspec
+      - run:
+          name: lint
```

Comment:
shall we lint before we test?

Reply:
imo that gets annoying as failing tests are more important than style violations